### PR TITLE
Fix unit tests for Composer Installer

### PR DESCRIPTION
**Pull request by karyna-tsymbal-atwix**
This PR fixes failed unit tests for Composer Installer.

Should be merged after https://github.com/magento/magento-composer-installer/pull/32

### CODE REVIEW

File: tests/MagentoHackathon/Composer/Magento/PathMappingTranslationTest.php [hunk 1]
```diff
@@ -31,29 +31,32 @@ protected function setUp()
...
         $this->dm = $this->getMockBuilder('Composer\Downloader\DownloadManager')
                 ->disableOriginalConstructor()
                 ->getMock();
         $this->composer->setDownloadManager($this->dm);
+        $this->getMockBuilder('Composer\Repository\InstalledRepositoryInterface')
+            ->getMock();
 
-        $this->repository = $this->getMock('Composer\Repository\InstalledRepositoryInterface');
-        $this->io = $this->getMock('Composer\IO\IOInterface');
+//        $this->repository = $this->getMock('Composer\Repository\InstalledRepositoryInterface');
```
Comment (xmav) [inline tests/MagentoHackathon/Composer/Magento/PathMappingTranslationTest.php:57]: Please remove if it is not needed

File: composer.json [hunk 2]
```diff
@@ -2,16 +2,11 @@
     "name":"magento/magento-composer-installer",
     "description":"Composer installer for Magento modules",
     "keywords":[ "composer-installer", "magento"],
-    "minimum-stability":"stable",
+    "minimum-stability":"dev",
```
Comment (xmav) [inline composer.json:5]: should it be reverted ?

File: tests/MagentoHackathon/Composer/Magento/PathMappingTranslationTest.php (hunk 1 above)
Comment (karyna-tsymbal-atwix) [reply tests/MagentoHackathon/Composer/Magento/PathMappingTranslationTest.php]: removed

File: composer.json (hunk 2 above)
Comment (karyna-tsymbal-atwix) [reply composer.json]: hi @xmav , thanks for the note, reverted

File: composer.json [hunk 3]
```diff
@@ -3,15 +3,10 @@
     "description":"Composer installer for Magento modules",
     "keywords":[ "composer-installer", "magento"],
     "minimum-stability":"stable",
+    "test_version": "dev-master",
```
Comment (xmav) [inline composer.json:6]: I don't think this needed. (added by tests ?)

Comment (karyna-tsymbal-atwix) [reply composer.json:6]: I added this. It's needed for tests
magento-composer-installer/tests/MagentoHackathon/Composer/Magento/FullStack/AbstractTest.php:21
magento-composer-installer/tests/MagentoHackathon/Composer/Magento/FullStack/AbstractTest.php:59

`perl -pi -e \'s/"version"/"test_version"/g\' ./composer.json`

**Comment by xmav** [comment]
@magento import code to magento-trigger/magento-composer-installer

**Pull request merged**